spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigProperties.java

```diff
@@ -146,11 +147,11 @@ public void setEnabled(boolean enabled) {
             this.enabled = enabled;
         }
 
-        public int getDelay() {
+        public Duration getDelay() {
```

Comment:
Just to confirm, the framework will convert 10s or 10ms or 10m to the right `Duration` type (10 seconds, 10 milliseconds, 10 minutes respectively) automatically and correctly?

Reply:
Yes, it will. [Spring boot doc](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-conversion-duration)

```
ns for nanoseconds
us for microseconds
ms for milliseconds
s for seconds
m for minutes
h for hours
d for days
```

Reply:
Awesome! Thanks.